assets/xml/objects/object_skb.xml

```diff
@@ -0,0 +1,64 @@
+<Root>
+    <File Name="object_skb" Segment="6">
+        <!-- Stalchild skeleton -->
+        <Skeleton Name="gStalchildSkel" Type="Normal" LimbType="Standard" Offset="0x41F8"/>
+
+        <!-- Stalchild Limbs -->
+        <Limb Name="gStalchildRootLimb" Type="Standard" Offset="0x40C8"/>
+        <Limb Name="gStalchildWaistLimb" Type="Standard" Offset="0x40D4"/>
+        <Limb Name="gStalchildRoot2Limb" Type="Standard" Offset="0x40E0"/>
+        <Limb Name="gStalchildRightFemurLimb" Type="Standard" Offset="0x40EC"/>
+        <Limb Name="gStalchildRightShinLimb" Type="Standard" Offset="0x40F8"/>
+        <Limb Name="gStalchildRightFootLimb" Type="Standard" Offset="0x4104"/>
+        <Limb Name="gStalchildLeftFemurLimb" Type="Standard" Offset="0x4110"/>
+        <Limb Name="gStalchildLeftShinLimb" Type="Standard" Offset="0x411C"/>
+        <Limb Name="gStalchildLeftFootLimb" Type="Standard" Offset="0x4128"/>
+        <Limb Name="gStalchildRibcageLimb" Type="Standard" Offset="0x4134"/>
```

Comment:
I think "rib cage" is 2 words. Either way, should be consistent with the textures.